### ISSUE TITLE
add IPFire 

### ISSUE THREAD

**Issue by bx169** [maxnet/berryboot#232]
make your pi into a beast firewall! http://downloads.ipfire.org/release/ipfire-2.17-core95 lmk know if you can add it.

**Comment by Dimensional** [comment]
Nice. Can be useful if you are using a 3G/4G internet connect connected to your USB port, or if you use your Pi as a wireless router with the ethernet connecting to the internet directly. Because it only has 1 ethernet port, it technically isn't useful. Not without wifi, Mobile Broadband, or a USB-to-LAN adapter.

**Comment by bx169** [comment]
yes i know was actully going to make it act like a router, just like a little firewall.
